src/brod.erl

```diff
@@ -740,6 +748,40 @@ start_link_topic_subscriber(Client, Topic, Partitions,
                                    ConsumerConfig, MessageType,
                                    CbModule, CbInitArg).
 
+%% @doc create topic(s) in kafka
+%% Return the message body of `create_topics, response.
+%% See `kpro_schema.erl' for struct details
```

Comment:
Consider using `@equiv` edoc tag [1] to avoid duplicating the docstring. Same applies to the other new functions in this module

[1] http://erlang.org/doc/apps/edoc/chapter.html#function-tags